src/validators/Room/RoomCreateBodyValidator.ts

```diff
@@ -1,17 +1,23 @@
 import {
   IsNotEmpty,
   IsString,
+  IsBoolean,
+  IsOptional,
 } from "class-validator";
 import { RoomCreateBody } from "../../types/Room/RoomCreateRequest";
 
 /**
    * Validate room create request body
    */
 export default class RoomCreateBodyValidator implements RoomCreateBody {
-  [x: string]: string;
+  [x: string]: string | boolean;
```

Comment:
The same as in other comments, no need for this

Reply:
I had to let this how it is right now due to a conflict generated in the RoomController due to the missing index signature.

Reply:
Were you not able to modify `RoomCreateBody` so that it doesn't extend `Record<string, string | boolean >` ?